bump_pydantic/codemods/validator.py

```diff
@@ -155,7 +166,12 @@ def _decorator_with_leading_comment(self, node: cst.Decorator, comment: str) ->
     def _replace_validators(self, node: cst.Decorator, old_name: str, new_name: str) -> cst.Decorator:
         RemoveImportsVisitor.remove_unused_import(self.context, "pydantic", old_name)
         AddImportsVisitor.add_needed_import(self.context, "pydantic", new_name)
-        decorator = node.decorator.with_changes(func=cst.Name(new_name), args=self._args)
+        if m.matches(node.decorator, m.Call()):
+            decorator = node.decorator.with_changes(func=cst.Name(new_name), args=self._args)
+        elif m.matches(node.decorator, m.Name()):  # @root_validator (Name, not Call) -> @model_validator(...) (Call)
+            decorator = cst.Call(func=cst.Name(new_name), args=self._args)
+        else:
+            raise RuntimeError("decorator as attribute is not implemented yet.")  # this should never happen ATM
```

Comment:
We shouldn't have this conditional. Can you add a xfail test for which this would happen?

Reply:
OK. Removed `RuntimeError` and moved the `Name` case under `else`.

> Can you add a xfail test for which this would happen?

I think the test is already there in [test_import_pydantic](https://github.com/pydantic/bump-pydantic/blob/03c4d319a4f7a2754130153e92834c8ea119dc06/tests/unit/test_validator.py#L281-L282), no?